addons/website_event/models/event_event.py

```diff
@@ -400,9 +400,9 @@ def _google_map_link(self, zoom=8):
 
     def _track_subtype(self, init_values):
         self.ensure_one()
-        if 'is_published' in init_values and self.is_published:
+        if 'website_published' in init_values and self.is_published:
             return self.env.ref('website_event.mt_event_published')
```

Comment:
Maybe we could take the opportunity to use raise_if_not_found=False and use the "note" subtype if it has been removed, to be defensive.

Reply:
Ok for the `raise_if_not_found` . But I'm not sure about the `note` . If the custom subtype has been deleted we want to default to the standard behavior no? So let `_track_subtype` return `False` and in the end use a log message.

But you have the final word and it's a simple change so say the word and i will add the "note" fallback.